packages/terra-form-field/src/terra-dev-site/test/form-field/FieldInline.test.jsx

```diff
@@ -35,6 +35,42 @@ const FieldExamples = () => (
     >
       <div style={{ border: 'dashed 1px lightGrey' }}>Control Placeholder</div>
     </Field>
+    <br />
+    <Field
+      id="inline-4"
+      style={{ border: 'dashed 1px lightGrey', padding: '10px' }}
+      label="Field Label"
+      isInline
+      error="Error Message"
+      isInvalid
+      required
+    >
+      <div style={{ border: 'dashed 1px lightGrey' }}>Control Placeholder</div>
+    </Field>
+
+    <Field
+      id="inline-5"
+      style={{ border: 'dashed 1px lightGrey', padding: '10px' }}
+      label="Field Label"
+      isInline
+      error="Error Message"
+      isInvalid
+      showOptional
+    >
+      <div style={{ border: 'dashed 1px lightGrey' }}>Control Placeholder</div>
+    </Field>
+
+    <Field
```

Comment:
What's the diffference between inline-4 and inline-6? They look the same at first glance, which we should use the same if they are.

Reply:
[Updated.](https://github.com/cerner/terra-core/pull/1537/commits/89ffbfe150e43b31ee6d3fc6fd022d0610b247d7#diff-210f2643e18d38a08a5296e9e1557a38R9)